app/styles/_settings.scss

```diff
@@ -21,6 +21,7 @@ $font-size-huge: 100px;
 // Font weights
 $font-weight-light: 300;
 $font-weight-normal: 400;
+$font-weight-semi: 600;
```

Comment:
Don't add this variable if we don't load the this font-weight from Google (see the first two lines of `_settings.scss`).